intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_files.h

```diff
@@ -211,9 +211,20 @@ struct model_file_loader {
     file.read_raw(&hparams.alibi_bias_max, sizeof(float));
     file.read_raw(&hparams.clip_qkv, sizeof(float));
     hparams.par_res = file.read_u32();
+
     hparams.word_embed_proj_dim = file.read_u32();
     hparams.do_layer_norm_before = bool(file.read_u32());
+    
+    // For ChatGLM-1 & 2
+    hparams.bos_token_id = file.read_u32();
+    hparams.eos_token_id = file.read_u32();
+    hparams.pad_token_id = file.read_u32();
+    hparams.sep_token_id = file.read_u32();
+    hparams.multi_query_group_num = file.read_u32();
```

Comment:
Is it for `MQA (multi-query-attention)`?

Reply:
Yes

Reply:
we already have `n_head_kv` param for `MQA`. Is it the same?

Reply:
I will check later. MQA has been recored in the TODO of this PR.